# Exceptions can be raised with arguments that give detail about them.
# For example:
name = "123"
raise NameError("Invalid name!")

# Traceback (most recent call last):#
#     raise NameError("Invalid name!")
# NameError: Invalid name!

